### PR TITLE
fix: bump mermaid version to `9.4.0` fixes #153

### LINKED ISSUE #153: Mermaid diagrams of type mindmap are not rendered
Hi @mjbvz - Thanks for your extension!

Mermaid now also [supports mindmaps](https://mermaid-js.github.io/mermaid/#/mindmap), a feature originally requested in #59 .

Unfortunately vscode-markdown-mermaid doesn't seem to render `mindmap` mermaid diagrams ight now, while `gitGraph` works. 

Is it just a missing mermaid dependency update?

### PR DESCRIPTION
**Pull request by gozes**
bumping the mermaid version to support mindmap rendering

will fix issue #153

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "markdown-mermaid",
-  "version": "1.17.4",
```
Comment (mjbvz) [inline package.json:3]: I will handle this for release. Should not be included in the PR

**Review by mjbvz — CHANGES_REQUESTED** [review]
Can you please also update the markdown version mentioned in the readme

**Comment by gozes** [comment]
@mjbvz both issues fix :)

**Review by mjbvz — APPROVED** [review]

**Comment by mjbvz** [comment]
Thanks!

**Pull request merged**